tools/stress-cluster/cluster/README.md

```diff
@@ -112,6 +113,14 @@ in an Azure AKS cluster (shared or personal).
 kind create cluster
 ```
 
+# Deploying Stress Test Addons
+Steps for deploying the stress test addons helm chart:
+1. Increment the version number in [Chart.yaml](https://github.com/Azure/azure-sdk-tools/blob/main/tools/stress-cluster/cluster/kubernetes/stress-test-addons/Chart.yaml) (e.g. 0.1.0 -> 0.1.1).
+1. Run [deploy.ps1](https://github.com/Azure/azure-sdk-tools/blob/main/tools/stress-cluster/cluster/kubernetes/stress-test-addons/deploy.sh).
+1. Update all the helm chart versions for stress-test-addons dependency references in [stress-cluster/chaos/examples/](https://github.com/Azure/azure-sdk-tools/tree/main/tools/stress-cluster/chaos/examples)**/Chart.yaml.
+   1. Please verify that the tests have succeeded with each pod to reaching Running/Completed state.
+1. Run azure-sdk-tools\eng\common\scripts\stress-testing\deploy-stress-tests.ps1 script in the [examples](https://github.com/Azure/azure-sdk-tools/tree/main/tools/stress-cluster/chaos/examples) directory, this will update all the nested helm charts (-login tag is needed for the first run).
```

Comment:
This step should probably be combined with the above sub-step, as it's the explicit instruction for what the above step says to do at a high level. Also I would show the command to verify everything is working (e.g. `kubectl get pods -n examples -w` and look for Running/Completed no Error, etc.)